Sources/Danger/BitBucketCloud.swift

```diff
@@ -0,0 +1,228 @@
+import Foundation
+
+public struct BitBucketCloud: Decodable {
+    /// The activities such as OPENING, CLOSING, MERGING or UPDATING a pull request
+    public let activities: [BitBucketCloudActivity]
+
+    /// The comments on the pull request
+    public let comments: [BitBucketCloudComment]
+
+    /// The commits associated with the pull request
+    public let commits: [BitBucketCloudCommit]
+
+    /// The pull request and repository metadata
+    public let metadata: BitBucketMetadata
+
+    /// The PR metadata
+    public let pr: BitBucketCloudPR
+}
+
+public struct BitBucketCloudPR: Decodable {
+    public enum State: String, Decodable {
+        case declined = "DECLINED"
+        case merged = "MERGED"
+        case open = "OPEN"
+        case suspended = "SUPERSEDED"
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case author
+        case createdOn = "created_on"
+        case description
+        case destination
+        case id
+        case participants
+        case reviewers
+        case source
+        case state
+        case title
+        case updatedOn = "updated_on"
+    }
+
+    /// The creator of the PR
+    public let author: BitBucketCloudUser
+
+    /// Date when PR was created
+    public let createdOn: Date
+
+    /// The text describing the PR
+    public let description: String
+
+    /// The PR's destination
+    public let destination: BitBucketCloudMergeRef
+
+    /// PR's ID
+    public let id: Int
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 55;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'id' (`identifier_name`)